src/graphql/event.rs

```diff
@@ -25,13 +29,116 @@ use std::{
     num::NonZeroU8,
     sync::{Arc, Mutex},
 };
-use tracing::warn;
+use tokio::time;
+use tracing::{error, warn};
 
 const DEFAULT_CONNECTION_SIZE: usize = 100;
+const EVENT_FETCH_TIME: u64 = 60 * 30;
```

Comment:
Please set the default `EVENT_FETCH_TIME` value to 20 seconds and consider to read this value from the config.toml